flake.nix

```diff
@@ -157,15 +163,18 @@
                 overlays = [ overlayGHC concat.overlay.${system} ]
                   ++ fullOverlays;
                 inherit system;
+                config.allowBroken = true;
```

Comment:
Could you add a comment explaining which broken package we’re trying to allow here?

I really wish `allowBroken` could be applied to derivations rather than the entire config.